eng/WpfArcadeSdk/tools/Packaging.targets

```diff
@@ -301,13 +302,19 @@ $(PreparePackageAssetsDependsOn):
       <RootFolderContents Include="$(ArtifactsPackagingDir)$(NormalizedPackageName)\*.*"/>
     </ItemGroup>
 
+    <PropertyGroup>
+      <DestinationRuntimeSubFolder Condition="'$(Platform)'=='AnyCPU' or '$(Platform)'=='x86' or '$(Platform)'=='Win32'">runtimes\win-x86\native\</DestinationRuntimeSubFolder>
+      <DestinationRuntimeSubFolder Condition="'$(Platform)'=='x64'">runtimes\win-x64\native\</DestinationRuntimeSubFolder>
+      <DestinationRuntimeSubFolder Condition="'$(Platform)'=='arm64'">runtimes\win-arm64\native\</DestinationRuntimeSubFolder>
```

Comment:
Why do we redefine this if `IdentifyPackageAssets` already does this? Or should this be hoisted into a common target?

Reply:
I believe these paths represent different things:  non-runtime specific binaries (`DestinationSubFolder`) and runtime-specific binaries (`DestinationRuntimeSubFolder`).  `DestinationSubFolder` should only be set to the same value as `DestinationRuntimeSubFolder` for native components, not for all components.

Reply:
I misread. LGTM on this part.

Reply:
Thanks.  I'm pushing one more change with the TFM update (win10-arm64 -> win-arm64).

Reply:
GitHub is incredibly sensitive about the co-author formatting.  Those lines must be flush left and at the end of the commit message or they are ignored.